cmd/celestia-appd/cmd/addrbook.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/p2p/pex"
+)
+
+const (
+	// routabilityStrict is a hard-coded config value for the address book.
+	// See https://github.com/celestiaorg/celestia-core/blob/793ece9bbd732aec3e09018e37dc31f4bfe122d9/config/config.go#L540-L542
+	routabilityStrict = true
```

Comment:
[Suggestion] Is it possible to reference the config setting or constant that stores the value of `AddrBookStrict`? Alternatively, can we retrieve it from the default P2P configuration? This would help ensure that the code in this section remains consistent with any future changes to this configuration.

Reply:
<img width="361" alt="Screenshot 2023-10-20 at 4 18 34 PM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/1468758d-5545-4fd5-b52f-0a49c140db99">

I don't see a constant exposed from celestia-core for this value

Reply:
How about using [this function](https://github.com/celestiaorg/celestia-app/blob/dfd99595cdecc9d50a5b363cfaf1829b23a5cf02/app/default_overrides.go#L230) i.e., `DefaultConsensusConfig().P2P.AddrBookStrict` to get the P2P config defaults? assuming that holds the correct default.